runtime/src/test/resources/application-test.yml

```diff
@@ -18,6 +18,10 @@ keycloak:
   enabled: true
   configurationFile: classpath:/ipaas-test-client.json
 
+endpoints:
```

Comment:
wonder whether we should think about introducing spring-boot profiles ? That way a profile can be easily selected during startup via an env (which we could add to the corresponding openshift template)